packit/local_project.py

```diff
@@ -5,6 +5,7 @@
 import git
 import requests
 
+from giturlparse import parse
```

Comment:
I just checked if the library is present in Fedora and sadly, it's not - that's a blocker for us: packit needs to run from RPM on a Fedora system:
```
$ dnf search giturlparse
No matches found.
```